dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp

```diff
@@ -62,6 +62,8 @@
     Object inodeObj =(Object) request.getAttribute("inode");
     String inode = inodeObj != null ? inodeObj.toString() : "";
 
+    String contentletIdentifierGlobal = contentlet.getIdentifier();
```

Comment:
This jsp uses the contentletIdentifier in multiple places. I would pull the `String contentletIdentifier = contentlet.getIdentifier();` out the Story Block segment (line 215) and declare it global, or modified that line to use your declared variable. I simply have one global with the contentlet identifier

Reply:
Good catch

Reply:
Done!